### DISCUSSION TITLE
Upgrade from 2.6 to 2.8 - per repo migration steps

### DISCUSSION THREAD

**Discussion by Mais316** [goharbor/harbor#20671] [Q&A]
Hi All,

Need your help to figure out per project chartmuseum upgrade steps to OCI, can see tool here https://github.com/goharbor/harbor/blob/main/tools/migrate_chart and I see it auto pushes to all repos by default how to control it to test per one project and verify all is Ok.

Can project admin use it only for his repos?

**Comment by MinerYang** [comment]
It is not supported by this tool.
